python2.7/run/runtime-mock.py

```diff
@@ -1,41 +1,148 @@
 from __future__ import print_function
 import sys
+import os
+import random
+import uuid
+import time
+import resource
+import datetime
 
 orig_stdout = sys.stdout
 orig_stderr = sys.stderr
 
-# TODO: finish this
+def eprint(*args, **kwargs):
+    print(*args, file=sys.stderr, **kwargs)
+
+def _random_account_id():
+  return random.randint(100000000000, 999999999999)
```

Comment:
Will need to consistent with spacing – I personally prefer 2 spaces, but was sticking to PEP-8 which uses 4